content/en/admin/install.md

```diff
@@ -116,6 +117,10 @@ CREATE USER mastodon CREATEDB;
 
 Done!
 
+### making change in the .ruby-version file:
+
+(note: In root login: edit the file in /home/mastodon/live/.ruby-version, enter the version installed, save and close it).
```

Comment:
You usually don't need to do this? We distirbute a .ruby-version file with the "current" version of Ruby we expect. For new installations, there is 0 reason to change it. Only if you want to keep using an old version is there any reason to modify this file.